packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -8,7 +8,48 @@
 import cx from 'classnames';
 import { usePrefix } from '../../internal/usePrefix';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { PropsWithChildren } from 'react';
+import { AccordionProvider } from './AccordionProvider';
+
+interface AccordionProps {
+  /**
+   * Specify the alignment of the accordion heading
+   * title and chevron. Defaults to `end`.
+   */
+  align?: 'start' | 'end';
+
+  /**
+   * Specify an optional className to be applied to
+   * the container node.
+   */
+  className?: string;
+
+  /**
+   * Specify whether an individual AccordionItem
+   * should be disabled.
+   */
+  disabled?: boolean;
+
+  /**
+   * Specify whether Accordion text should be flush,
+   * default is `false`, does not work with `align="start"`.
+   */
+  isFlush?: boolean;
+
+  /**
+   * Specify the size of the Accordion. Currently
+   * supports the following: `sm`, `md`, `lg`
+   */
+  size?: 'sm' | 'md' | 'lg';
+}
+
+interface AccordionV10Props extends Omit<AccordionProps, 'size'> {
+  /**
+   * Specify the size of the Accordion. Currently
+   * supports the following: `sm`, `md`, `lg`, `xl`.
+   */
+  size?: 'sm' | 'md' | 'lg' | 'xl';
+}
```

Comment:
Is there a reason we are bringing back the v10 `xl` size here?

Reply:
I was just trying to be backwards compatible, but I will remove this